### DISCUSSION TITLE
Spare parts multiplies in vehicle cargo

### DISCUSSION THREAD

**Discussion by AshTrayBY** [Vdauphin/HeartsAndMinds#1546] [Question and answer] [answered]
Hi all,
not sure it's related to H&M but I'm having following issue - after server save and restart all btc_vehicles have their spare parts in cargo multiplied. And after 2-3 more server restarts they have zero cargo space and no spare parts in cargo.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
You need to disable the ACE setting adding spare parts in cargo
Cheers

**Comment by AshTrayBY** [comment]
Thanks, seems it helps.
